code/game/objects/items/cigs_lighters.dm

```diff
@@ -490,6 +490,15 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	light_color = LIGHT_COLOR_FIRE
 	grind_results = list("iron" = 1, "welding_fuel" = 5, "oil" = 5)
 
+/obj/item/lighter/suicide_act(mob/living/carbon/user)
+	if (lit)
+		user.visible_message("<span class='suicide'>[user] begins holding \the [src]'s flame up to [user.p_their()] face! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+		playsound(src, 'sound/items/welder.ogg', 50, 1)
+		return FIRELOSS

```

Comment:
Should self ignite or something

Reply:
i was thinking about that, but I was worried about the possibility of starting fires with something that cant start fires (even in a roundabout way)